ch.sbb.scion.rcp.microfrontend.app.demo/.project

```diff
@@ -21,6 +21,13 @@
 			</arguments>
 		</buildCommand>
 	</buildSpec>
+	<linkedResources>
+		<link>
+			<name>.settings</name>
+			<type>2</type>
+			<locationURI>PARENT-1-PROJECT_LOC/ch.sbb.scion.rcp.eclipse.setup/settings</locationURI>
```

Comment:
Shouldn't this commit contain also the "ch.sbb.scion.rcp.eclipse.setup" project and the preset improvements? (you forgot to stage those, or I'm missing something here?)

Reply:
of course. Eclipse put my bundle in the wrong directory. Changed that and pushed the missing bundle